client/Makefile

```diff
@@ -37,11 +36,18 @@ start-frontend:
 .PHONY: smoke-test
 smoke-test:
 	start_server_and_test \
-		'../launch_dev_server.sh --config-file $(CXG_CONFIG) --port $(CXG_SERVER_PORT)' \
+		'SKIP_ATAC_CACHE=true ../launch_dev_server.sh --config-file client/__tests__/e2e/test_config.yaml --port $(CXG_SERVER_PORT)' \
 		$(CXG_SERVER_PORT) \
 		$(if $(and $(CURRENT_SHARD),$(TOTAL_SHARDS)), \
-			'CXG_URL_BASE="http://localhost:$(CXG_SERVER_PORT)" npm run e2e -- --shard=$(CURRENT_SHARD)/$(TOTAL_SHARDS)', \
-			'CXG_URL_BASE="http://localhost:$(CXG_SERVER_PORT)" npm run e2e')
+			'CXG_URL_BASE="http://localhost:$(CXG_SERVER_PORT)" npm run e2e -- --project=chromium --shard=$(CURRENT_SHARD)/$(TOTAL_SHARDS)', \
+			'CXG_URL_BASE="http://localhost:$(CXG_SERVER_PORT)" npm run e2e -- --project=chromium')
+
+.PHONY: data-persistence-smoke-tests
```

Comment:
Adds data-persistence-smoke-tests which boot the server with the persistence config and runs the 'data-persistence' Playwright project.